### ISSUE TITLE
crop_image.py

### ISSUE THREAD

**Issue by gmyenni** [weecology/EvergladesTools#16]
Do ya'all need this? 
https://github.com/weecology/EvergladesWadingBird/blob/main/DataCleaningScripts/crop_image.py

**Comment by bw4sz** [comment]
no, it does appear in any search results for this or the other repo. Can delete.
